src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Stream.cs

```diff
@@ -100,8 +103,17 @@ public Http2Stream(HttpRequestMessage request, Http2Connection connection, int s
             private object SyncObject => _streamWindow;
 
             public int StreamId => _streamId;
-            public HttpRequestMessage Request => _request;
-            public HttpResponseMessage Response => _response;
+
+            public HttpResponseMessage GetAndClearResponse()
```

Comment:
Nit: I wonder if the Response property could now be removed or made private, to avoid against accidentally retrieving this without clearing the reference.

Reply:
> I wonder if the Response property could now be removed or made private

I'm not following... I did remove the Response property (and the Request property).  Is there another one you're referring to?

Reply:
Nope, I just missed it, sorry!